packages/http-client-csharp/emitter/src/lib/operation-converter.ts

```diff
@@ -207,6 +207,7 @@ function fromSdkHttpOperationParameter(
     Kind: getParameterKind(p, parameterType, rootApiVersions.length > 0),
     DefaultValue: getParameterDefaultValue(p.clientDefaultValue, parameterType),
     Decorators: p.decorators,
+    SkipUrlEncoding: (p as SdkPathParameter).allowReserved === true
```

Comment:
How about other type such as `SdkQueryParameter`?

Reply:
Query parameter doesn't have the concept of allowReserved.